src/Parsers/ASTExpressionList.cpp

```diff
@@ -37,10 +37,8 @@ void ASTExpressionList::formatImplMultiline(const FormatSettings & settings, For
         {
             if (separator)
                 settings.ostr << separator;
-            settings.ostr << ' ';
```

Comment:
FWIW `TSVRaw` uses `formatImpl` and it will still has trailing whitespaces

Reply:
Ok, I will fix it also.